### ISSUE TITLE
remote get should fail on macOS

### ISSUE THREAD

**Issue by morgo** [datacharmer/dbdeployer#76]
**Describe the bug**

Remote get "seems to work" on macOS.  There is a usability bug, where if someone doesn't read the instructions they can find themselves in a scenario that is not straight forward to debug.

**To Reproduce**

```
$ dbdeployer remote download mysql-5.7.25
File /Users/xxx/mysql-5.7.25.tar.xz downloaded
$ dbdeployer unpack mysql-5.7.25.tar.xz
Unpacking tarball mysql-5.7.25.tar.xz to $HOME/Devel/mysql/5.7.25
.........100.........200.........300.........400.........500.........600.........700.........800.........900.........1000.........1100.........1200.........1300.........1400.........1500.........1600.........1700.........1800.........1900.........2000.........2100.........2200.........2300.........2400.........2500.........2600.........2700.........2800.2811
Renaming directory /Users/xxx/Devel/mysql/mysql-5.7.25 to /Users/xxx/Devel/mysql/5.7.25
$ dbdeployer deploy single 3.0.0 --client-from=5.7.25
Database installed in $HOME/Devel/sandboxes/msb_3_0_0
run 'dbdeployer usage single' for basic instructions'
. sandbox server started
err: exit status 126
cmd: /Users/xxx/Devel/sandboxes/msb_3_0_0/load_grants
stdout:
error creating sandbox: 'exit status 126'
```

**Expected behavior**

I think it should fail at this step:
```
$ dbdeployer remote download mysql-5.7.25
Error: remote download is currently only supported on Linux
```
You could add a `--force` if you really wanted to allow people to overwrite this.

**Environment:**
 - OS: macOS
 - dbdeployer version: It was > 1.19
 - tarball full name (e.g. mysql-5.6.40-linux-glibc2.12-x86_64.tar.gz)
 - tarball origin (from where it was downloaded) 

**Hardware: (if applicable)**
 - Free storage
 - Total RAM

**Additional context**

This happened to a colleague, but I confess it happened to me once too when I wasn't thinking.  So I thought it was worth reporting.

**Comment by datacharmer** [comment]
Thanks for this report.
Yes, it's easy to get mixed up with this command.
I am going to add a check on the operating system and fail graciously when it is not Linux.
If I manage to add support for MacOS binaries, I will disable it.

**Comment by datacharmer** [comment]
The behavior will change as follows:
```
$ dbdeployer remote get mysql-8.0.15
The binaries retrieved by this command are only for Linux
Set the environment variable 'DBDEPLOYER_FORCE_REMOTE_GET' to force a download on a non-Linux host
```
When forcing the download, we get:
```
$ DBDEPLOYER_FORCE_REMOTE_GET=1 dbdeployer remote get mysql-8.0.15
Variable 'DBDEPLOYER_FORCE_REMOTE_GET' was set. Forcing download on OS 'darwin'
################################################################################
# WARNING: the binaries being downloaded WILL NOT WORK on the current OS.
################################################################################
File /Users/gmax/go/src/github.com/datacharmer/dbdeployer/mysql-8.0.15.tar.xz downloaded
```

**Comment by morgo** [comment]
LGTM

**Comment by datacharmer** [comment]
Fixed in release 1.26.0
